src/routes/api/collabs/[id]/picks/[pick_id]/delimage.ts

```diff
@@ -0,0 +1,110 @@
+import type { RequestHandler } from '@sveltejs/kit';
```

Comment:
Instead of /delimage, name it just /image, which is a bit neater since we already do a DELETE call to this endpoint. Renaming to /image allows us to put more calls in here later as well, suct as image uploads, which have their own route rn.

Reply:
did that